### PR TITLE
Shanoir issue#1593: change password

### PR DESCRIPTION
**Pull request by pierrehenri-dauvergne**
Logged in users can now update their password from the profile page with the "update password" button

### CODE REVIEW

**Comment by michaelkain** [comment]
Hi Julien (@julien-louis), it would be great to review this PR this week please? Thank you and with kind regards, Michael

File: docker-compose/keycloak/themes/shanoir-theme/login/login-update-password.ftl [hunk 1]
```diff
@@ -12,6 +12,7 @@
                 <div class="${properties.kcInputWrapperClass!}">
                     <input type="password" id="password-new" name="password-new" class="${properties.kcInputClass!}"
                            autofocus autocomplete="new-password"
+						   style="color: red;"
```
Comment (julien-louis) [inline docker-compose/keycloak/themes/shanoir-theme/login/login-update-password.ftl:15]: Why is that for ? 
image

**Review by pierrehenri-dauvergne — COMMENTED** [review]
Fixed password color

**Comment by michaelkain** [comment]
Hi Julien (@julien-louis), hi Pierre-Henri (@pierrehenri-dauvergne), it would be nice to bring that feature live? Happy to know, if you might be able to integrate it e.g. for a test session tomorrow morning? With kind regards, Michael

**Review by julien-louis — APPROVED** [review]

**Pull request merged**